pkg/liqoctl/install/install_test.go

```diff
@@ -0,0 +1,254 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"reflect"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/action"
+
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
+	installprovider "github.com/liqotech/liqo/pkg/liqoctl/install/provider"
+	installutils "github.com/liqotech/liqo/pkg/liqoctl/install/utils"
+)
+
+type StringInterfaceMap map[string]interface{}
+
+// IsContainedIn checks whether specMap's key-value pairs are equal to testMap's ones (not the other way around).
+func (specMap StringInterfaceMap) IsContainedIn(testMap map[string]interface{}) bool {
+	isContained := false
+
+	for key := range specMap {
+		testValue, ok := testMap[key]
+		if !ok {
+			break
```

Comment:
Would `return false`, to make it clear the intent. Do not rely on side-effects, since they might no longer be true after future code changes. Same below.